jumpscale/packages/threebot_deployer/chats/threebot.py

```diff
@@ -94,7 +97,14 @@ def deploy(self):
                 **self.solution_metadata,
             )
         )
-        success = deployer.wait_workload(self.workload_ids[0], self)
+        deploying_message = f"""
+# Deploying your 3Bot...\n\n
+<br>It will usually take a few minutes to succeed. Please wait patiently.\n
```

Comment:
`take a few minutes to finish` and `redirected to the next step once successful` maybe?